proto/ligato/kvscheduler/value_status.go

```diff
@@ -35,7 +33,7 @@ func (m ValueStatus) MarshalJSON() ([]byte, error) {
 func (m *ValueStatus) UnmarshalJSON(data []byte) error {
 	return jsonpb.Unmarshal(bytes.NewReader(data), m)
 }
-
+*/
```

Comment:
Can't decide whether this should be removed or refactored using protoV2 tooling, but it probably should not stay commented out. The problems is that i can't find usage of this code to decide that. There is no direct usage, but as it implements json.Marshaller/Unmarshaller interface, it can be called dynamically somewhere.